### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by Blacksheep70**
Little tweaks to include helicopter stats

### CODE REVIEW

File: php/cron_classes.php [hunk 1]
```diff
@@ -39,24 +39,36 @@ function addNewEntrys($mysqli) {
...
-		$mysqli->query($query6);
-		$mysqli->query($query7);
+		$query6 = "INSERT INTO pilot_aircrafts (pilot_aircrafts.pilotid, pilot_aircrafts.aircraftid) SELECT DISTINCT pilots.id, aircrafts.id FROM dcs_events, pilots, aircrafts WHERE dcs_events.InitiatorPlayer = pilots.name AND dcs_events.InitiatorType = aircrafts.name AND dcs_events.InitiatorGroupCat = 'HELICOPTER' AND aircrafts.id NOT IN (SELECT pilot_aircrafts.aircraftid FROM pilot_aircrafts WHERE pilot_aircrafts.pilotid = pilots.id )";
+		$query7 = "INSERT INTO pilot_aircrafts (pilot_aircrafts.pilotid, pilot_aircrafts.aircraftid) SELECT DISTINCT pilots.id, aircrafts.id FROM dcs_events, pilots, aircrafts WHERE dcs_events.TargetPlayer = pilots.name AND dcs_events.TargetType = aircrafts.name AND dcs_events.TargetGroupCat = 'HELICOPTER' AND aircrafts.id NOT IN (SELECT pilot_aircrafts.aircraftid FROM pilot_aircrafts WHERE pilot_aircrafts.pilotid = pilots.id )";
+		$mysqli->query($query9);
+		$mysqli->query($query10);
+		$mysqli->query($query11);
+		$mysqli->query($query12);
```
Comment (mahaupt) [inline php/cron_classes.php]: $query9 - 12 were not defined

**Review by mahaupt — CHANGES_REQUESTED** [review]
The variable names are messed up a bit, we should probably call the query function directly instead of saving every query into a variable.

**Review by mahaupt — COMMENTED** [review]
Alright. I fixed it (I think). Now it needs testing

**Comment by Blacksheep70** [comment]
Works for me

**Pull request merged**